ajax/libs/FlexSearch/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "FlexSearch",
+  "description": "Next-Generation full text search library with zero dependencies.",
+  "author": "Thomas Wilkerling",
+  "license": "Apache-2.0",
+  "keywords": [
+    "search",
+    "fulltext",
+    "fastest",
+    "contextual",
+    "fuzzy",
+    "matching",
+    "scoring",
+    "indexing",
+    "fieldsearch"
+  ],
+  "filename": "",
```

Comment:
Let's set the default file here.

Reply:
Done. Could anyone squash the commits please? Thanks.